File: MOBILE/ebanking_mb/src/store/LoginResponse.ts
interface LoginResponse {
  id: number;
  username: string;
  fullName: string;
  address: string;
  citizenId: string;
  birthday: string;
  createAt: string;
  isMale: string;
  roles: Set<string> | string[];
  jwt: string;
  ekycStatus?: 'NOT_VERIFIED' | 'VERIFIED' | 'EXPIRED' | 'REJECTED';
  pinStatus?: boolean;
  privateKey: any;
}

File: BACKEND/authService/src/main/java/com/example/auth/dto/response/LoginResponse.java
package com.example.auth.dto.response;

import lombok.Builder;
import lombok.Data;

import java.security.PrivateKey;
import java.util.Set;

@Data
@Builder
public class LoginResponse {
    private Long id;
    private String username;
    private String fullName;
    private String address;
    private String citizenId;
    private String birthday;
    private String createAt;
    private boolean isMale;
    private Set<String> roles;
    private String jwt;
    String privateKey;
}
